### PR TITLE
Hide important section when no important messages

### PR DESCRIPTION
**Pull request by GretaD**
Fixes parts of #6142

### CODE REVIEW

File: src/components/MailboxThread.vue [hunk 1]
```diff
@@ -22,9 +22,9 @@
 					:bus="bus" />
 				<template v-else>
 					<div class="app-content-list-item">
-						<SectionTitle v-if="hasImportantEnvelopes" class="important" :name="t('mail', 'Important')" />
+						<SectionTitle v-if="!hasImportantEnvelopes" class="important" :name="t('mail', 'Important')" />
```
Comment (GretaD) [inline src/components/MailboxThread.vue:25]: hmm, this is still shown :weary:

**Comment by arjentz** [comment]
Just wondering, is this one actually not a draft?

**Comment by GretaD** [comment]
> Just wondering, is this one actually not a draft?

hey arjen, thanks a lot for having a look. This is suppose to work, but the labels are not hidden when theres no messages on important section, and i dont know what im doing wrong. Thats why i kept it in draft.

File: src/components/MailboxThread.vue [hunk 2]
```diff
@@ -280,6 +286,9 @@ export default {
 .app-content-list-item:hover {
 	background: transparent;
 }
+app-content-list-item {
```
Comment (ChristophWurst) [inline src/components/MailboxThread.vue:289]: missing a dot?

**Comment by ChristophWurst** [comment]
@GretaD please test again

File: src/components/EmptyMailboxSection.vue [hunk 3]
```diff
@@ -1,7 +1,5 @@
 <template>
-	<div id="emptycontent">
-		<slot> {{ t('mail', 'Messages marked as important will show up here') }} </slot>
```
Comment (ChristophWurst) [inline src/components/EmptyMailboxSection.vue:3]: I believe this file might be obsolete now

Comment (ChristophWurst) [reply src/components/EmptyMailboxSection.vue:3]: I don't think that is the case. `EmptyMailboxSection` renders when `is-priority-inbox=true`, which is only done for the two priority inbox sections in `MailboxThread`. And we no longer show the *important* mailbox when it's empty, only *Other*.

So whenever *Other* is empty, we show a display-none'd `EmptyMailboxSection`.

This means we have to undo the display none and rectify `Messages marked as important will show up here` to make sense for *Others*. It was a wrong assumption that the component would only ever show for important emails.

Comment (GretaD) [reply src/components/EmptyMailboxSection.vue:3]: when i switch between mailboxes, this is shown
Screenshot from 2022-10-17 16-18-39

Comment (GretaD) [reply src/components/EmptyMailboxSection.vue:3]: exactly, that was my problem since the beginning i cannot hide the label

Comment (ChristophWurst) [reply src/components/EmptyMailboxSection.vue:3]: Test again. I accidentally undid one of my changes when I committed. Fixed now.

Comment (GretaD) [reply src/components/EmptyMailboxSection.vue:3]: yes, all looks good now

**Review by ChristophWurst — APPROVED** [review]

**Pull request merged**